src/current/_includes/v23.1/sidebar-data/feature-overview.json

```diff
@@ -0,0 +1,82 @@
+{
+    "title": "Feature Overview",
+    "is_top_level": true,
+    "items": [{
+        "title": "CockroachDB Products",
+        "urls": [
+          "/${VERSION}/choose-a-deployment-option.html"
+        ]
+      },
+      {
+        "title": "CockroachDB Basics",
+        "items": [
+          {
+            "title": "Why CockroachDB?",
+            "urls": [
+              "/${VERSION}/why-cockroachdb.html"
+            ]
+          },
+          {
+            "title": "Replication & Rebalancing",
+            "urls": [
+              "/${VERSION}/demo-replication-and-rebalancing.html"
+            ]
+          },
+          {
+            "title": "Fault Tolerance & Recovery",
+            "urls": [
+              "/${VERSION}/demo-fault-tolerance-and-recovery.html"
+            ]
+          },
+          {
+            "title": "Serializable Transactions",
+            "urls": [
+              "/${VERSION}/demo-serializable.html"
+            ]
+          },
+          {
+            "title": "Multi-Active Availability",
+            "urls": [
+              "/${VERSION}/multi-active-availability.html"
```

Comment:
This needs to link to the relevant section. When I click the link, I just get the Availability FAQ.

Reply:
Updated the name of the page to be "Multi-Active Availability"

Reply:
Thanks.